lib/content-services/src/lib/search/components/search-filter-chips/search-filter-chips.component.ts

```diff
@@ -29,7 +29,7 @@ import { FacetField, SearchCategory, TabbedFacetField } from '../../models';
     encapsulation: ViewEncapsulation.None
 })
 export class SearchFilterChipsComponent implements OnInit, OnDestroy {
-    private queryBuilder = inject(SearchQueryBuilderService);
+    queryBuilder = inject(SearchQueryBuilderService);
```

Comment:
This is no longer required as `queryBuilder` is not used in HTML, please make it private again